src/porepy/fracs/msh_2_grid.py

```diff
@@ -230,7 +253,7 @@ def create_1d_grids(
     tol: float = 1e-4,
     constraints: Optional[np.ndarray] = None,
     return_fracture_tips: bool = True,
-):
+) -> tuple[list[pp.Grid], np.ndarray] | list[pp.Grid]:
```

Comment:
If I'm not misreading the return declaration, it does not seem to match the documentation.